composer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "igorescobar/jquery-mask-plugin",
```

Comment:
Change it to jquery-mask-plugin. And I will setup the web hook. Already got an account. @igorescobar.